README.md

```diff
@@ -1,2 +1,103 @@
+	- Linking Type: OAuth - Authorization Code
+	- Client ID: this is your Client Id from Google API Console ('OAuth 2.0 Client IDs' click your webclient credentials)
+	- Client secret: this is your Client Secret from Google API Console ('OAuth 2.0 Client IDs' click your webclient credentials)
+	- Authorization URL - https://XXXXXXXXX.herokuapp.com/oauth
+	- Token URL - https://XXXXXXXXX.herokuapp.com/oauth/token
+	- Scopes (I added google-home-authcode / app-user because i saw those 2 somewhere in the code but I don't think they are needed) ?? @Andrei please confirm/deny
+- Back in Overview click Add Actions
+- Fulfillment URL - https://XXXXXXXXX.herokuapp.com/smarthome/fulfill
+
+
+## CONFIG.TS
+
+ Pull the repo, and modify [src/config.ts](https://github.com/andrei-tatar/nora-service/blob/master/src/config.ts)
+
+    export const oauthClientId = 'api console - credentials - oauth 2.0 client ids - web client - Client ID';
+    export const oauthClientSecret = 'api console - credentials - oauth 2.0 client ids - web client - Client secret';
+    export const googleProjectApiKey = 'api console - credentials - api keys - key';
+    export const serviceAccount = {
+        project_id: 'downloaded service account json from firebase',
+        client_email: 'downloaded service account json from firebase',
+        private_key: 'downloaded service account json from firebase'
+    };
+
+    apiKey: 'firebase - settings -general - your apps',
+    authDomain: 'firebase - settings -general - your apps',
+    databaseURL: 'firebase - settings -general - your apps',
+    projectId: 'firebase - settings -general - your apps',
+    storageBucket: 'firebase - settings -general - your apps',
+    messagingSenderId: 'firebase - settings -general - your apps',
+
+
+## USER.REPOSITORY.TS
+Modify [src/services/user.repository.ts](https://github.com/andrei-tatar/nora-service/blob/master/src/services/user.repository.ts)
+
+Not  sure if there is another way ([Andrei](https://github.com/andrei-tatar) please confirm/deny) but to create the Database Tables I had to modify this lines
+
+    (async function () {
+        const service = new PostgressService();
+        await service.query(`
+            CREATE TABLE IF NOT EXISTS appuser (
+                uid VARCHAR(30) CONSTRAINT pk PRIMARY KEY,
+                linked boolean DEFAULT false
+            )`
+        );
+    
+        await service.query('ALTER TABLE appuser ADD COLUMN IF NOT EXISTS noderedversion integer DEFAULT 1');
+    	await service.query('ALTER TABLE appuser ADD COLUMN IF NOT EXISTS refreshtoken integer DEFAULT 1');
+    })().catch(err => {
+        console.error(err);
+    }).then(() => {
+        console.log('done');
+    });
+
+
+## PUSH
+
+I think THAT'S IT, you are ready to deploy to Heroku! 
+I would suggest to build the app and the  schema (just to check we didn't mess up [Andrei's](https://github.com/andrei-tatar) code :grimacing:) with `npm run build && npm run schema` before deploying.
```

Comment:
You can actually run `npm run heroku-postbuild` and it will do the code build, json schema and copy the html files to the dist folder.